python/hail/matrixtable.py

```diff
@@ -2386,9 +2386,9 @@ def format_type(typ):
             '----------------------------------------\n' \
             'Entry fields:{e}\n' \
```

Comment:
I changed the `describe` format to
```
>>> mt.describe()
----------------------------------------
Global fields:
    None
----------------------------------------
Column fields:
    'col_idx': int32
----------------------------------------
Row fields:
    'row_idx': int32
----------------------------------------
Entry fields:
    None
----------------------------------------
Column key: ['col_idx']
Row key: ['row_idx']
Partition key: ['row_idx']
----------------------------------------
```
The way the field strings are created, they begin with a newline, so there's no need for a space. The key strings on the other hand are now single-line lists. I did this so that you can tell the difference between `Key: []` and `Key: None`.